simphony/cuds/abc_modeling_engine.py

```diff
@@ -16,6 +16,8 @@ class ABCModelingEngine(object):  # pragma: no cover
         container of attributes related to the computational method
     SP : DataContainer
         container of attributes related to the system parameters/conditions
+    materials : Materials
```

Comment:
I think this is a good way to proceed. I don't see a point in using too much time on this, because there will be a change coming to it anyway.